firmware/controllers/map_averaging.cpp

```diff
@@ -170,6 +178,18 @@ static void endAveraging(void *arg) {
 #if EFI_PROD_CODE || defined(__DOXYGEN__)
 	v_averagedMapValue = adcToVoltsDivided(
 			mapAccumulator / mapMeasurementsCounter);
+	if (mapMinBufferLength > 1) {
```

Comment:
Please be aware that this code is inside lockAnyContext()! It's all about optimization (the feature is rarely used).